guides/common/modules/con_puma-workers-and-threads-recommendation.adoc

```diff
@@ -0,0 +1,16 @@
+[id="Puma_Workers_and_Threads_Recommendation_{context}"]
+= Puma Workers and Threads Recommendations
+
+In order to recommend thread and worker configurations for the different tuning profiles, we conducted Puma tuning testing on {Project} with different tuning profiles and the main test run performed in this testing was concurrent registration with the following combinations along with different workers and threads.
+Our recommendation is based purely on concurrent registration performance, so it might not reflect your exact use-case.
+For example, if your setup is very content oriented with lots of publishes and promotes, you might want to limit resources consumed by Puma in favor of Pulp and PostgreSQL.
+
+[width="100%",cols="16%,19%,7%,7%,31%,20%",options="header",]
+|===
+|Name |Number of managed host |RAM |Cores |Recommended Puma Threads for both min & max |Recommended Puma Workers
+|default |0-5000 |20GiB |4 |16 |4-6
+|medium |5000-10000 |32GiB |8 |16 |8-12
+|large |10000-20000 |64GiB |16|16 |12-18
+|extra-large |20000-60000 |128GiB |32 |16 |16-24
+|extra-extra-large |60000+ |256GiB+ |48+ |16 |20-26
```

Comment:
Hello Ewoud. That is what we measured - IIRC there was some slight improvement with more threads - we have "Our recommendation is based purely on concurrent registration performance, so it might not reflect your exact use-case." there.